README.rst

```diff
@@ -98,13 +98,21 @@ Quick Example
 
     bot = commands.Bot(command_prefix='$')
 
-    @bot.command()
-    async def ping(ctx):
-        await ctx.reply('Pong!')
+
+    @bot.slash_command(description="Replies with pong!")
+    async def ping(interaction: nextcord.Interaction):
+        await interaction.send("Pong!")
```

Comment:
I think having 1 example command would be enough. Maybe remove the "ephemeral" command, and instead keep just the "ping" command but make the "Pong!" response ephemeral.